src/components/Chatlog.js

```diff
@@ -0,0 +1,37 @@
+import React from 'react';
+import ChatEntry from './ChatEntry';
+import PropTypes from 'prop-types';
+import './ChatLog.css';
+
+const ChatLog = ({ entries, updateLiked }) => {
+  const chatLogComponents = entries.map((chatEntry) => {
+    return (
+      <ChatEntry
+        key={chatEntry.id}
+        id={chatEntry.id}
+        sender={chatEntry.sender}
+        body={chatEntry.body}
+        timeStamp={chatEntry.timeStamp}
+        liked={chatEntry.liked}
+        updateLiked={updateLiked}
+      ></ChatEntry>
+    );
+  });
+
+  return <div>{chatLogComponents}</div>;
```

Comment:
This likely doesn't need its own `div`, we may be able to use React fragments (`<>` and `</>`) instead.